### PR TITLE
Fix/improvements

### PR DESCRIPTION
**Pull request by levanion**

### CODE REVIEW

File: apps/ui/src/pages/Datasource/constants.ts [hunk 1]
```diff
@@ -2,30 +2,30 @@ export const DATA_LOADER_IMAGES = [
   {
     sourceName: 'Postgres',
     imageSrc:
-      'https://res.cloudinary.com/practicaldev/image/fetch/s--twVcwT-B--/c_imagga_scale,f_auto,fl_progressive,h_900,q_auto,w_1600/https://dev-to-uploads.s3.amazonaws.com/i/kbjzeples71613068b1i.jpg',
+      'https://upload.wikimedia.org/wikipedia/commons/thumb/2/29/Postgresql_elephant.svg/800px-Postgresql_elephant.svg.png',
```
Comment (Chkhikvadze) [inline apps/ui/src/pages/Datasource/constants.ts:5]: Download that mages in our assets and import from there

File: apps/ui/src/pages/Datasource/constants.ts [hunk 2]
```diff
@@ -2,30 +2,30 @@ export const DATA_LOADER_IMAGES = [
...
     sourceName: 'MySQL',
-    imageSrc: 'https://logowik.com/content/uploads/images/mysql8604.logowik.com.webp',
+    imageSrc:
+      'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcStwXG33lFYL4EOU3XEdsb0gT3e14xyOuwC4A&usqp=CAU',
   },
   {
     sourceName: 'Upload File',
     imageSrc:
-      'https://w7.pngwing.com/pngs/382/24/png-transparent-computer-icons-symbol-windows-10-symbol-miscellaneous-angle-rectangle-thumbnail.png',
+      'https://static.vecteezy.com/system/resources/previews/012/325/824/original/file-dark-mode-glyph-ui-icon-attach-document-digital-information-user-interface-design-white-silhouette-symbol-on-black-space-solid-pictogram-for-web-mobile-isolated-illustration-vector.jpg',
```
Comment (Chkhikvadze) [inline apps/ui/src/pages/Datasource/constants.ts:15]: Download that mages in our assets and import from there

File: apps/ui/src/pages/Datasource/constants.ts [hunk 3]
```diff
@@ -2,30 +2,30 @@ export const DATA_LOADER_IMAGES = [
...
   },
   {
     sourceName: 'Web Page',
     imageSrc:
-      'https://e7.pngegg.com/pngimages/272/463/png-clipart-computer-icons-web-development-globe-internet-globe-miscellaneous-emblem.png',
+      'https://icon-library.com/images/website-icon-png-transparent/website-icon-png-transparent-9.jpg',
```
Comment (Chkhikvadze) [inline apps/ui/src/pages/Datasource/constants.ts:20]: Download that mages in our assets and import from there

File: apps/ui/src/pages/Datasource/constants.ts [hunk 4]
```diff
@@ -2,30 +2,30 @@ export const DATA_LOADER_IMAGES = [
...
   },
   {
     sourceName: 'Notion',
     imageSrc:
-      'https://seuladogeek.com.br/wp-content/uploads/2023/01/notion-logo-vector-and-png.png',
+      'https://images-wixmp-ed30a86b8c4ca887773594c2.wixmp.com/f/b86edfa2-993d-4f6a-87e4-d0709fd9c0b1/ddgxeer-4754f58f-17b1-4d33-b8e3-b158a10a9aba.png/v1/fill/w_1280,h_1280/white_notion_logo_icon_for_black_dark_theme_by_seyahdoo_ddgxeer-fullview.png?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJ1cm46YXBwOjdlMGQxODg5ODIyNjQzNzNhNWYwZDQxNWVhMGQyNmUwIiwiaXNzIjoidXJuOmFwcDo3ZTBkMTg4OTgyMjY0MzczYTVmMGQ0MTVlYTBkMjZlMCIsIm9iaiI6W1t7ImhlaWdodCI6Ijw9MTI4MCIsInBhdGgiOiJcL2ZcL2I4NmVkZmEyLTk5M2QtNGY2YS04N2U0LWQwNzA5ZmQ5YzBiMVwvZGRneGVlci00NzU0ZjU4Zi0xN2IxLTRkMzMtYjhlMy1iMTU4YTEwYTlhYmEucG5nIiwid2lkdGgiOiI8PTEyODAifV1dLCJhdWQiOlsidXJuOnNlcnZpY2U6aW1hZ2Uub3BlcmF0aW9ucyJdfQ.cumez1G4ONxtJEtwuYDFMWxdL-VUReQXufSovBCVI14',
```
Comment (Chkhikvadze) [inline apps/ui/src/pages/Datasource/constants.ts:25]: Download that mages in our assets and import from there

File: apps/ui/src/pages/Datasource/constants.ts [hunk 5]
```diff
@@ -2,30 +2,30 @@ export const DATA_LOADER_IMAGES = [
...
   },
   {
     sourceName: 'Shopify',
-    imageSrc:
-      'https://play-lh.googleusercontent.com/PS_kNe60lcXXULOOAwAe58Ti-i1ByKrLBprWaTXd_FTXkk-6sUoaxqMtE8ZfYp07Ng=w600-h300-pc0xffffff-pd',
+    imageSrc: 'https://assets.stickpng.com/images/58482ec0cef1014c0b5e4a70.png',
```
Comment (Chkhikvadze) [inline apps/ui/src/pages/Datasource/constants.ts:29]: Download that mages in our assets and import from there

**Review by Chkhikvadze — CHANGES_REQUESTED** [review]

File: apps/ui/src/pages/Settings/useSettings.ts [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+        value: values.open_api_key,
+        account_id: currentAccount.id,
+        is_secret: true,
+        is_required: true,
+      }
+      const huggingFaceTokenValues = {
+        key: 'hugging_face_token',
+        key_type: 'string',
+        value: values.hugging_face_token,
+        account_id: currentAccount.id,
```
Comment (okradze) [inline apps/ui/src/pages/Settings/useSettings.ts:48]: i think you should not send account id from frontend.
You have access to account id in backend auth

File: apps/ui/src/services/config/useUpdateConfigService.ts [hunk 7]
```diff
@@ -18,10 +28,11 @@ export const useUpdateConfigService = () => {
           is_secret,
           is_required,
           datasource_id: datasource_id,
-          team_id : team_id,
+          team_id: team_id,
           agent_id: null,
           toolkit_id: tool_id,
           workspace_id: null,
+          account_id,
```
Comment (okradze) [inline apps/ui/src/services/config/useUpdateConfigService.ts:35]: use account id in config controller backend

**Pull request merged**